CHANGELOG.md

```diff
@@ -19,10 +19,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: New trick, Cross Bomb Launch.
 - Changed: The Shinesink Clip and Aim Down Clip tricks are now a single Floor Clip trick.
 
+##### Artaria
+
+- Added: Video: Ballsparking into Speed Hallway from the right, charging speed from Energy Recharge Station South.
+- Changed: Doors into Charge Beam Access are excluded from being randomized into Grapple Beam Doors, in order to fascilitate Ballsparking into Speed Hallway.
```

Comment:
This change isn't needed. you can still get the ballspark off easily if both of them are grapple doors.